Dockerfile.base

```diff
@@ -90,9 +90,9 @@ RUN apt-get install -y nano vim graphviz && \
     pip3 install nose2 jupyterlab
 
 # Install commons
-RUN git clone https://github.com/atlab/commons.git && \
-    pip3 install commons/python && \
-    rm -r commons
+# RUN git clone https://github.com/vathes/commons.git && \
```

Comment:
Why is this commented out? Also you probably want to make this point to `reimerlab` rather than `vathes`. We are volume mounting to override anyway.

Reply:
For the moment, we are using the mount commons as the package. Paul is in favor of incorporating the relevant common tables in the current directory. After that, commons not needed it. We'll clean it up after Jake approves.

Reply:
i `pip3 install -e` my local copy of the `commons` repository, as we were modifying the table definitions.  we may bring over a few `commons` tables to `odor_meso`, in which case we would remove this dependency/installation altogether.